packages/groqd-playground/src/components/Playground.tsx

```diff
@@ -320,22 +326,27 @@ export default function GroqdPlayground({ tool }: GroqdPlaygroundProps) {
 
       return (
         <Flex flex={1} direction="column">
-          <Box marginY={3} paddingX={3}>
-            <Label muted>Error</Label>
-          </Box>
-          <Box
-            paddingX={3}
-            paddingY={1}
-            style={{ maxHeight: 250 }}
-            overflow="auto"
-            marginBottom={2}
-          >
-            {errorView}
-          </Box>
-          <Box paddingX={3} marginY={3}>
-            <Label muted>Raw Response {execTimeDisplay}</Label>
-          </Box>
-          <JSONExplorer data={rawResponse} highlightedPaths={errorPaths} />
+          <Split mode="vertical">
+            <Flex direction="column" style={{ maxHeight: 400 }}>
+              <Box marginY={3} paddingX={3}>
+                <Label muted>⚠️ Error</Label>
```

Comment:
Not to be a nit, but isn't the ⚠️ more for warnings? Maybe a 🚫 ?

Reply:
"Warning, there's an error" 😆